deepinv/optim/optim_iterators/gradient_descent.py

```diff
@@ -28,7 +29,7 @@ def __init__(self, **kwargs):
         self.f_step = fStepGD(**kwargs)
         self.requires_grad_g = True
 
-    def forward(self, X, cur_data_fidelity, cur_prior, cur_params, y, physics):
+    def forward(self, X, cur_data_fidelity, cur_prior, cur_params, y, physics, *args):
```

Comment:
Since `OptimIterator` has **kwargs in its forward, would it make sense to add it here too? (and in every itreration)